### DISCUSSION TITLE
googlezip.net gws US 这个类型的ip 是否可用。

### DISCUSSION THREAD

**Discussion by wpyok500** [SeaHOH/GotoX#77] [点子]
googlezip.net gws US 这个类型的ip 是否可用。

**Comment by SeaHOH** [comment]
好像不能吧，给我个试试。

**Comment by wpyok500** [comment]
不好意思昨天，发完贴好，睡下了。
[ip_tmpok.txt](https://github.com/SeaHOH/GotoX/files/1981823/ip_tmpok.txt)

**Comment by wpyok500** [comment]
如果这类型的ip可用，那就太好了，这类型的ip好像不少。

**Comment by SeaHOH** [comment]
不能用。
其实我也正在扫谷歌的 IPv4，已经 30 多个小时了，大概还有 10 多个小时就能完成。

**Comment by wpyok500** [comment]
@SeaHOH  到时来一份，先拜谢了。

**Comment by SeaHOH** [comment]
总共才扫出不到两百，不过有几个速度很快。
以下是我能扫出的范围，稍微扩大了点，毕竟只是单个样例而已。
50 线程用了 50 小时扫谷歌官方 IPv4，有点慢，但是不显眼。
```
64.233.160.0/24
64.233.167.0/24
64.233.179.0/24
74.125.0.0/16
108.177.9.0/24
108.177.111.0/24
136.22.128.0/22
172.217.0.0/16
173.194.50.0/24
173.194.67.0/24
173.194.73.0/24
173.194.79.0/24
173.194.146.0/24
173.194.150.0/24
173.194.163.0/24
173.194.175.0/24
173.194.195.0/24
173.194.210.0/24
173.194.211.0/24
208.65.152.0/24
209.85.192.0/18
216.58.192.0/18
```

**Comment by wpyok500** [comment]
谢谢，小白请教下：
64.233.160.0/24是指64.233.160.1-4.233.160.255
74.125.0.0/16是指74.125.0.1-74.125.255.255
请问136.22.128.0/22  和 209.85.192.0/18 是指？

**Comment by SeaHOH** [comment]
IP 是一个数字，`/` 后面的数字就是指 IP 的前缀位数，和 IP 一起标识一个网络，跟网络掩码意思差不多，扫描软件基本上都能解析。

**Comment by SeaHOH** [comment]
已经完蛋了，估计是谷歌方面做的（无责任猜想）。

**Comment by SeaHOH** [comment]
那是官方 IP，不是 SNI。

**Comment by SeaHOH** [comment]
关键是有个 GGC 也败了，我没发过这个啊。

**Comment by SeaHOH** [comment]
这好像是 bug，不过启用 verifyg2pk 后是不会触发的。

**Comment by SeaHOH** [comment]
所以才有验证，不然。。。

**Comment by SeaHOH** [comment]
等会儿修复这错误。

**Comment by Just2co** [comment]
@SeaHOH verify2pk关掉不用验证公钥，这样性能会好一些，速度快一些？我这个想法正确不？还有你上面给的那些IP段能否扫除SNI？

**Comment by SeaHOH** [comment]
建议开启公钥验证，这点开销可忽略。

至于 IP，上面说了是谷歌官方 IP，并非 SNI。
而且已经完蛋了，包括未列出的 GGC，至少我这里是。

**Comment by yuli6800** [comment]
请问，用gsacn扫sni时，用默认的servername:www.bing.coms扫出来的可以用吗？还有扫出来的ip是放在config里的【gae】下的iplist还是【iplist】下的google_2p，还是怎么改变设置

**Comment by SeaHOH** [comment]
看 https://github.com/SeaHOH/GotoX#使用 ，忽略对数量要求的描述。

**Comment by fredericky123** [comment]
@Just2co 是否有sni ip库推荐呢？我这边找不到一个可用的
